### ISSUE TITLE
New feature request: Dockerfile

### ISSUE THREAD

**Issue by Exceter007** [midwire/free_zipcode_data#20]
It would be considerably easier to run this very useful tool if there was a Dockerfile in the repository which could be used to create and populate the sqlite database.

**Comment by midwire** [comment]
Yeah this is a good idea. As usual, I'm swamped with my paying job, but I'll do what I can to get to it.  Any PR would be appreciated in the meantime.
